### ISSUE TITLE
Travis fail 

### ISSUE THREAD

**Issue by dorianps** [dorianps/LESYMAP#3]
@muschellij2 ,

I am compiling Lesymap on Travis based on the script you prepared sometime ago:
http://bit.ly/antsr_travis_sh

but it fails:
https://travis-ci.org/dorianps/LESYMAP/jobs/244766485

Is `http://bit.ly/antsr_travis_sh` the most recent script to test packages that depend on ANTsR (it may have outdated info like `ants_user=stnava`)?

Any other suggestion how to resolve Travis fails? Here is the current travis file for Lesymap:
https://github.com/dorianps/LESYMAP/blob/master/.travis.yml

Thank you.

**Comment by dorianps** [comment]
The problem was specific to failure on installing RcppArmadillo.  I had some communication with the develpers of that package https://github.com/RcppCore/RcppArmadillo/issues/163. Turns out the problem is only when clang was used to compile, and it might be related to parameters set up in our antsr travis helpers. The code in those helpers also forces clang to be used instead of g++. I resolved the issue by calling the installation of RcppArmadillo before everything else, and this allowed RcppArmadillo to be installed successfully with g++. 

@muschellij2  , was there a particular reason why you chose force clang to be used in travis ignoring the `compiler` flag in the travis file, or is this a bug?

Closing the issue since the problem is temporarily resolved.
